boot/zephyr/single_loader.c

```diff
@@ -221,6 +221,12 @@ read_image_size(const struct flash_area *fa_p,
 }
 
 
+
+/* Get the SOC's flash erase block size from the DTS, fallback to 1024. */
+#define DT_DRV_COMPAT soc_nv_flash
+#define SOC_FLASH_ERASE_BLK_SZ \
+         DT_PROP_OR(DT_INST(0, soc_nv_flash), erase_block_size, 1024)
```

Comment:
better to use the chosen "zephyr_flash" to get the flash properties?

`DT_PROP_OR(DT_CHOSEN(zephyr_flash), erase_block_size,1024)`